tests/libs/util/hash.cpp

```diff
@@ -14,7 +14,7 @@
 
 _hash::_hash(const std::string& algorithm)
 {
-    HRESULT hr;
+    long hr;
```

Comment:
I think I'd let _this_ specific instance be as-is. HRESULT is very windows specific and given that there are macros provided that operate on this type, I'd leave this specific type alone.  Maybe the more experienced windows folks could weigh in?

Reply:
Agree, leave HRESULT.